app/views/layouts/application.html.erb

```diff
@@ -9,23 +9,80 @@
   </head>
 
   <body>
+    <nav class="navbar navbar-default">
+      <div class="container-fluid">
+        <!-- Brand and toggle get grouped for better mobile display -->
+        <div class="navbar-header">
+          <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1" aria-expanded="false">
+            <span class="sr-only">Toggle navigation</span>
+            <span class="icon-bar"></span>
+            <span class="icon-bar"></span>
+            <span class="icon-bar"></span>
+          </button>
+          <%= link_to "Chairman Meow", root_path, class: 'navbar-brand' %>
+        </div>
 
-    <%= link_to "Home", meow_products_path %>
-    <% if user_signed_in? || admin_signed_in? %>
-      <%= link_to "Edit Profile", edit_user_registration_path(@user) %>
-      <%= link_to "Log Out", destroy_user_session_path, :method => "delete", data:{ confirm:"ARE YOU SURE?!?" } %>
-    <% else %>
-      <%= link_to "New User", new_user_registration_path %>
-      <%= link_to "User Log In", new_user_session_path %>
-      <%= link_to "Admin Log In", new_admin_session_path %>
+        <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
+          <ul class="nav navbar-nav">
+            <!-- <li class="active"><a href="#">Link <span class="sr-only">(current)</span></a></li> -->
+            <li class="dropdown">
+              <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Our selection<span class="caret"></span></a>
+              <ul class="dropdown-menu">
+                <li><%= link_to 'Newest additions', newest_meow_products_path %></li>
+                <li><%= link_to 'Premium breeds', premium_meow_products_path %></li>
+                <li role="separator" class="divider"></li>
+                <li><%= link_to 'Browse all cats', meow_products_path %></li>
+              </ul>
+            </li>
+          </ul>
+          <form class="navbar-form navbar-left">
+            <div class="form-group">
+              <%= form_tag search_meow_products_path, method: :get do %>
+                <%= text_field_tag :query, params[:query], placeholder: 'Looking for a specific cat?', class: 'form-control' %>
+                <%= submit_tag 'Search', class: 'btn btn-default' %>
+                <!-- <input type="text" name="query" class="form-control" placeholder="Looking for a specific cat?" autocomplete="off"> -->
```

Comment:
Here's the searchbar! When I hit search, instead of going to `search_meow_products_path`, the path just goes to `http://localhost:3000/?utf8=%E2%9C%93&query=LOOKFORCAT&commit=Search` :cry: Any help would be greatly appreciated! Thanks!